pkg/altbn128/altbn128.go

```diff
@@ -297,3 +297,14 @@ func (e *gfP2) pow(base *gfP2, exp *big.Int) *gfP2 {
 	}
 	return e
 }
+
+func UnmarshalG2Point(bytes []byte) (*G2Point, error) {
+	g2 := new(bn256.G2)
+
+	_, err := g2.Unmarshal(bytes)
+	if err != nil {
+		return nil, err
+	}
+
+	return &G2Point{g2}, nil
+}
```

Comment:
I am not sure if we should have this implementation here. `altbn128.go` implements additional functions that are not available in `bn256` and `Unmarshal` is there. Anyway, please look at my next comment, I don't think we need this function here anyway.